src/main/kotlin/com/beust/kobalt/plugin/publish/PublishPlugin.kt

```diff
@@ -133,7 +134,13 @@ class PublishPlugin @Inject constructor(val files: KFiles, val factory: PomGener
                         messages.add(taskResult.errorMessage!!)
                     }
                 }
-                git.maybeTagRelease(project, TaskResult(), configuration.autoGitTag)
+                if (autoGitTagConfig != null) {
```

Comment:
But of duplication here with the same code in `Git.kt`, think it could be factored out?